### ISSUE TITLE
Generic Product Fallback Info

### ISSUE THREAD

**Issue by dhulihan** [hulihanapplications/fletcher#11]
When someone tries to fetch product information from a URL that isn't supported by a model, fletcher should return generic fallback information composed of the following:
- name: based on page title (`meta` tag)
- price: search page for first `$...` string
- images: all images on page.

**Comment by davidcollom** [comment]
wouldn't all images on the page be a little excessive? also what about other currencies? GBP, EUR, YEN....?  cross selling or related items may be higher in the source then the actual item/product price.

agreed with the title, this should be implemented, maybe fall back to meta description of exists?

**Comment by dhulihan** [comment]
Agreed, it's hard to pull relevant images, so that may be out. Pulling description from meta is a good idea.

As far as price goes, we could also check for standardized meta information, like the the presence of [Open Graph Protocol](http://ogp.me/) tags, etc. Just a thought. 

Have you played with the [amazon wishlist bookmarklet](http://www.amazon.com/wishlist/get-button)? They're able to parse an arbitrary price pretty accurately.

**Comment by davidcollom** [comment]
Agreed on opg, brilliant idea. amazon are possibly passing things into a readability and then scanning for currency symbols?

**Comment by oyeanuj** [comment]
@dhulihan @hulihanapplications Any updates on this ticket? This would be invaluable given the prolificity of stores today?
